src/index.tsx

```diff
@@ -1,13 +1,10 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import firebase from "firebase/app";
+import * as React from "react";
```

Comment:
Refer to https://stackoverflow.com/questions/55285737/import-as-react-from-react-vs-import-react-from-react.
I also set `"allowSyntheticDefaultImports": false` in the [tsconfig](https://github.com/digidem/mapeo-webmaps/pull/68/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0)